it/uk/gov/hmrc/pushpullnotificationsgateway/connectors/ConnectorSpec.scala

```diff
@@ -14,12 +14,13 @@
  * limitations under the License.
  */
 
-package uk.gov.hmrc.pushpullnotificationsgateway.connector
+package uk.gov.hmrc.pushpullnotificationsgateway.connectors
 
+import org.mockito.{ArgumentMatchersSugar, MockitoSugar}
 import util._
 
 abstract class ConnectorSpec
-    extends AsyncHmrcSpec 
-    with NoMetricsGuiceOneAppPerSuite 
+    extends AsyncHmrcSpec
+    with NoMetricsGuiceOneAppPerSuite
     with WireMockSugar
-    with WireMockExtensions {}
+    with WireMockExtensions  with MockitoSugar with ArgumentMatchersSugar {}
```

Comment:
These shouldn't be needed with AsyncHmrcSpec

Reply:
OK will fix